scripts/setup-data-dir.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
```
#!/usr/bin/env bash
```
To keep consistent with other scripts. See a0bfbed92bcf89e4afeaaaf43ccbc9ed0afde49f